README.md

```diff
@@ -96,7 +96,7 @@ This token is currently given on the @sympy-bot GitHub user. To regenerate it,
 login as @sympy-bot, and go to https://github.com/settings/tokens/. Create a
 new token, checking only the `public_repo` box. Be sure to indicate in the
 description that the token is for Travis testing, and be sure to revoke any
-old tokens. The @sympy-bot user needs push access to this repo for the test to
+old tokens. The @sympy-bot user needs to push access to this repo for the test to
```

Comment:
The `to` shouldn't be here i think.

Reply:
yup got it thanks for the suggestion @certik